content/en/docs/how-tos/use-registries-in-build-farm.md

```diff
@@ -1,112 +1,243 @@
 ```
 
-Note that the only github user that has access to the image is the author of the underlying pull request which triggers the testrun.
+**Warning:** Only `vSphere` system administrators can access the images on [registry.apps.build01-us-west-2.vmc.ci.openshift.org](registry.apps.build01-us-west-2.vmc.ci.openshift.org).
+
+## How do I access the latest published images for my component?
 
-# Access the promoted images
+If the `ci-operator` configuration for your component configures image [`promotion`](/docs/architecture/ci-operator/#publishing-container-images),
+output container images will be published to the central CI registry when changes are merged to your repository. Two main
+configurations are possible for promotion: configuring an `ImageStream` name or a tag.
 
-If `ci-operator`'s config has [`promotion` stanza](/docs/architecture/ci-operator/#publishing-container-images), the images are published to the central CI registry at `app.ci`.
+### Publication of New Tags
+
+A configuration that specifies the `ImageStream` name looks like the following and results in new tags on that stream:
 
 ```yaml
 images:
-- dockerfile_path: images/hello-openshift/Dockerfile.rhel
+- dockerfile_path: images/my-component
   from: base
-  inputs:
-    ocp_builder_rhel-8-golang-1.15-openshift-4.7:
-      as:
-      - registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7
-  to: hello-openshift
+  to: my-component
 promotion:
   name: "4.7"
   namespace: ocp
 ```
 
-We can pull the image defined by above `promotion` stanza by:
+The `my-component` image can be pulled from the authoritative registry with:
 
 ```bash
-$ docker pull registry.ci.openshift.org/ocp/4.7:hello-openshift
+$ podman pull registry.ci.openshift.org/ocp/4.7:my-component
 ```
 
-# Troubleshooting
+### Publication of New Streams
+
+A configuration that specifies the `ImageStream` tag looks like the following and results in new streams in the namespace:
+
+```yaml
+images:
+- dockerfile_path: images/my-component
+  from: base
+  to: my-component
+promotion:
+  namespace: my-organization
+  tag: latest
+```
+
+The `my-component` image can be pulled from the authoritative registry with:
+
+```bash
+$ podman pull registry.ci.openshift.org/my-organization/my-component:latest
+```
+
+## Why I am getting an authentication error?
+
+An authentication error may occur both in the case where you have not yet logged in to a registry and in the case where
+you logged in to the registry in the past.
+
+### I have not yet logged in to the registry.
+
+Please follow [the directions](#how-do-i-log-in-to-pull-images-that-require-authentication) to log in to the registry.
+
+### I have logged in to the registry in the past.
+
+An unfortunate side-effect of the architecture for container image registry authentication results in authentication
+errors when your authentication token expired, even if the image you are attempting to pull requires no authentication.
```

Comment:
can we mention the expected token duration so people can anticipate it a bit?

Reply:
ACK - I think it's a month